stl/inc/generator

```diff
@@ -377,8 +377,9 @@ public:
         ++*this;
     }
 
-    _NODISCARD bool operator==(default_sentinel_t) const noexcept {
-        return _Coro.done();
+    _NODISCARD_FRIEND bool operator==(const _Gen_iter& _It, default_sentinel_t) noexcept /* strengthened */
+    {

```

Comment:
Another stylistic nitpick: There's enough room for the brace here to be attached; clang-format exhibits "sticky" behavior and will preserve it. Also not worth resetting testing for this PR.